docs/cloud-native-security/kspm.asciidoc

```diff
@@ -1,2 +1,38 @@
 [[kspm]]
 = Kubernetes security posture management
+
+The Kubernetes Security Posture Management (KSPM) integration allows you to monitor how your Kubernetes clusters' configuration measures up to security benchmarks.
+
+To set up the integration, you'll need to first add it to an agent policy, then deploy the KSPM DaemonSet to the Kubernetes clusters you want to monitor.
+
+[discrete]
+== Set up a KSPM integration
+To install the integration:
+
+* Go to *Security -> Dashboards -> Cloud Posture*
```

Comment:
```suggestion
. Go to *Security -> Dashboards -> Cloud Posture*.
```

Reply:
Not sure we need to tell users to navigate to Security. AFAIK, it's assumed they're already within the Security app.